pkg/kubelet/kubeletconfig/status/status.go

```diff
@@ -28,7 +28,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	clientset "k8s.io/client-go/kubernetes"
 	nodeutil "k8s.io/component-helpers/node/util"
-	"k8s.io/kubernetes/pkg/kubelet/metrics"
 )
 
 const (
```

Comment:
Config-related constants can probably be removed too.

Reply:
Actually can this entire file be deleted?

Reply:
there are some dependencies with kubeadm. I can do it in a follow up PR

Reply:
Ok.